lib/i18n/hygiene/key_usage_checker.rb

```diff
@@ -21,32 +19,22 @@ def fully_qualified_key_used?(key)
         if pluralized_key_used?(key)
           fully_qualified_key_used?(without_last_part(key))
         else
-          %x<git grep #{key} #{git_grep_options} | wc -l>.strip.to_i > 0
+          directories = @directories.join(" ")
+          include_argument = @file_extensions.map { |extension| "--include=\"*.#{extension}\"" }.join(" ")
+          exclude_argument = @exclude_files.map { |file| "--exclude=\"#{file}\""} .join(" ")
+
+          grep_command = [
+            "grep --recursive",
+            key,
+            include_argument,
+            exclude_argument,
+            directories
+          ].compact.join(" ")
```

Comment:
I think `reject(&:empty?)` may be better than `compact`. Compacting will only ever remove the `key` from the array if it is `nil`, which is not really a valid use-case.

Reply:
Agreed! Also removes unnecessary white spaces. Specs were adjusted as well!

Updated!